dev/com.ibm.ws.security.fat.common.backchannelLogout/fat/src/com/ibm/ws/security/backchannelLogout/fat/CommonTests/LogoutTokenValidationTests.java

```diff
@@ -515,8 +540,10 @@ public void LogoutTokenValidationTests_missing_required_iat() throws Exception {
     @Test
     public void LogoutTokenValidationTests_invalid_iat() throws Exception {
 
-        // set an invalid iat (future date - 2/21/2035)
-        genericInvalidClaimTest(Constants.PAYLOAD_ISSUED_AT_TIME_IN_SECS, 2055696852, MessageConstants.CWWKS1773E_TOKEN_EXPIRED);
+        long iat = 2055696852; // set an invalid iat (future date - 2/21/2035)
```

Comment:
If we're going to be updating this test case, I'd like to take the opportunity to do this programmatically.
```suggestion
        long iat = System.currentTimeMillis() / 1000 + hoursToSeconds(24 * 365 * 10);
```